docs/component/oa-verify.md

```diff
@@ -29,7 +29,7 @@ A verification happens on a wrapped document, and it consists of answering to so
 Before starting to play with the library, create a file `document.json` having the following content:
 ```json
 {
-  "version": "open-attestation/2.0",
+  "version": "https://schema.openattestation.com/2.0/schema.json",
```

Comment:
Hmm for TS, they might need to import this type. Not sure if you want to cover it here.

Reply:
No this guide is JS only :)

Reply:
if required I can upgrade all the code of all documents to typescript (I dont want to support both)